pkg/apis/edge/v1alpha1/logical-cluster.go

```diff
@@ -46,19 +46,24 @@ type LogicalCluster struct {
 // LogicalClusterSpec describes a cluster.
 type LogicalClusterSpec struct {
 	// ProviderName is the name of the provider of the cluster.
-	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="cluster is immutable"
+	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="ProviderName is immutable"
 	ProviderName string `json:"ProviderName"`
```

Comment:
Please add explanation that the ProviderName is used to identify the resources defined for this provider